stubs/resources/css/app.scss

```diff
@@ -55,26 +55,40 @@
 }
 
 // Dropdown
-.dropdown .dropdown__content {
+[aria-expanded] + ul {
+    display: none;
+}
+
+[aria-expanded="true"] + ul {
+    display: flex;
+}
+
+@media (min-width: 48rem) {
+    [aria-expanded] {
+        display: none;
+    }
```

Comment:
Some of these styles are very generic, for example this will hide other expandable elements on the page. Are you expecting all of these to be overridden with scoping as needed or perhaps the whole file replaced?

Reply:
This addresses a layout primitive, namely a button whose `aria-expanded` state is used to toggle the visibility of the subsequent element in the DOM. The stylesheet itself shows how this can be overridden with scoping for the mobile menu toggle:

```css
@media (min-width: 48rem {
    nav > [aria-expanded] + ul {
        display: flex;
    }
}
```

Anyone using Hearth will need to create their own stylesheet so I think it's appropriate to use this very generic style and add the above rule for the exception.